src/test/hardhatRPC.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+require("dotenv").config();
+import Web3 from "web3";
+
+let latestSnapshot: string;
+
+export const snapshotHardhat = async () => {
+  const provider = new Web3.providers.HttpProvider(process.env.RPC_URL as string);
+  const snapshotResponse: any = await new Promise((resolve, reject) =>
```

Comment:
Substitute `any` with `unknown`. [It's preferred in TypeScript now.](https://devblogs.microsoft.com/typescript/announcing-typescript-3-0-rc-2/#the-unknown-type)